plugins/vpc-bridge/config/k8s_connector.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+//go:build enablek8sconnector && windows
```

Comment:
`k8s` package is windows dependent. We are using `winio` for pipe communication and this is windows only package. since this file is dependent on `k8s` package, we have to add windows tag

Reply:
Makes sense.